javalib/src/main/scala-2/java/lang/annotation/RetentionPolicy.scala

```diff
@@ -1,3 +1,5 @@
+// Enums are not source-compatbile, make sure to sync this file Scala 3 implementation
```

Comment:
```suggestion
// Enums are not source-compatible, make sure to sync this file's Scala 3 implementation
```
Typo, here and in other scala-2 files